Sources/Controls/UIScrollView+Combine.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  UIScrollView+Combine.swift
+//  CombineCocoa
+//
+//  Created by Joan Disho on 09.08.19.
+//
+
+import Foundation
+import Combine
+
+public extension UIScrollView {
+    /// A publisher emitting content offset changes from this UIScrollView.
+    var contentOffsetPublisher: AnyPublisher<CGPoint, Never> {
+        publisher(for: \.contentOffset)
+            .eraseToAnyPublisher()
+    }
+
+    /// A publisher emitting if the bottom of the UIScrollView is reached.
+    ///- parameter offset: A threshhold indicating the bottom of the UIScrollView. Default 0.0.
```

Comment:
```suggestion
    /// - parameter offset: A threshold indicating the bottom of the UIScrollView. Default 0.0.
```

Can you refine this description? maybe a threshold indicating how close to the bottom edge of the scroll view is more correct here?